packages/influx-metrics-tracker/src/index.ts

```diff
@@ -0,0 +1,63 @@
+import { Logger } from '@ovotech/winston-logger';
+import { InfluxDB } from 'influx';
+
+export abstract class MetricsTracker {
+  constructor(
+    protected influx: InfluxDB,
+    protected logger: Logger,
```

Comment:
would it be worth making the `influx` and `logger` args optional too? Maybe we just want to pass in config

Reply:
Influx will need to be mandatory - it's where the metrics are being stored.

The logger could be made optional in the future if it's deemed necessary. For compatibility purposes, however, it's safer to start mandatory until required otherwise. This is since going from mandatory->optional is backwards-compatible, while optional->mandatory is not.